src/app/account-settings-page/account-settings-page.component.html

```diff
@@ -0,0 +1,12 @@
+<div class="block-user-table-container mat-elevation-z8">
```

Comment:
You should use a `mat-card` here instead of a div so that the elevation and background color matches the content of the other pages.